### PR TITLE
Config server doc updates

### PR DESCRIPTION
**Pull request by andytael**
Config Server Doc updates

### CODE REVIEW

File: docs-source/spring/content/platform/config/_index.md [hunk 1]
```diff
@@ -11,22 +11,393 @@ configuration in the Oracle Autonomous Database, so it easily supports labelled
 environments, as well as being accessible to a wide range of tools for managing the content.
 Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
```
Comment (markxnelson) [inline docs-source/spring/content/platform/config/_index.md:14]: feels like there is a word missing here?  to introduce the link

File: docs-source/spring/content/platform/config/_index.md [hunk 2]
```diff
@@ -11,22 +11,393 @@ configuration in the Oracle Autonomous Database, so it easily supports labelled
...
-  cloud:
-    config:
-      label: latest
-      profile: dev
 ```
 
-This example fetches data where the application is `atael`, profile is `dev` and the label is `latest`.
+Managing the data for the Spring Cloud Config server should be done using the CLI or via the REST API endpoints. If you prefer, you can also work directly with the `CONFIGSERVER.PROPERTIES` table in the database. ([Accessing the database](../../database/)).
+
+During setup of Oracle Backend for SPring Boot and Microservices the following data is loaded into `CONFIGSERVER.PROPERTIES`. This data can be deleted.
```
Comment (markxnelson) [inline docs-source/spring/content/platform/config/_index.md:30]: typo in SPring

File: docs-source/spring/content/platform/config/_index.md [hunk 3]
```diff
@@ -11,22 +11,393 @@ configuration in the Oracle Autonomous Database, so it easily supports labelled
...
+    "name": "atael",
+    "label": "",
+    "profile": ""
+  }
+]
+```
+
+#### /srv/config/all?service-profile=\<profile-name\>
+
+Get all distinct services filtered on profile(service-profile).
```
Comment (markxnelson) [inline docs-source/spring/content/platform/config/_index.md:109]: space?

File: docs-source/spring/content/platform/config/_index.md (hunk 2 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:30]: Fixed

File: docs-source/spring/content/platform/config/_index.md (hunk 1 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:14]: Rewrote the "init" section.

File: docs-source/spring/content/platform/config/_index.md (hunk 3 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:109]: Fixed

**Review by markxnelson — APPROVED** [review]

File: docs-source/spring/content/platform/config/_index.md [hunk 4]
```diff
@@ -1,32 +1,395 @@
...
-title: "Configuration"
+title: "Spring Config Server"
 ---
 
-Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized
-configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications
-across all environments.
+Oracle Backend for Spring Boot and Microservices includes Spring Cloud Config which provides server- and client-side support for externalized configurations in a distributed system. The Spring Cloud Config server provides a central place to manage external properties for applications across all environments.
 
 The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:7]: Change to:

... configured to store the following:

File: docs-source/spring/content/platform/config/_index.md [hunk 5]
```diff
@@ -1,32 +1,395 @@
...
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+configuration in the Oracle Autonomous Database, so it easily supports labeled versions of configuration environments, as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:8]: Change to:

Configuration in the Oracle Autonomous Database to support labeled versions of configuration environments as well as ...

File: docs-source/spring/content/platform/config/_index.md [hunk 6]
```diff
@@ -1,32 +1,395 @@
...
-  application:
-    name: atael
-  cloud:
-    config:
-      label: latest
-      profile: dev
 ```
 
-This example fetches data where the application is `atael`, profile is `dev` and the label is `latest`.
+Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table. Managing the data for the Spring Cloud Config server should be done using the CLI or via the REST API endpoints. If you prefer, you can also work directly with the `CONFIGSERVER.PROPERTIES` table in the database. How to access the database is documented here, ([Accessing the database](../../database/)).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:18]: Delete the term "via".

File: docs-source/spring/content/platform/config/_index.md [hunk 7]
```diff
@@ -1,32 +1,395 @@
...
+| application-b   | development    | 23cbeta  | db-dev-connection   | connectionstring-b-dev            |
+| application-c   | secret         | 23.4     | json-db             | 23c-json-db                       |
+| application-c   | secret         | 23.4     | json-sdb-conn       | 23c-mongo-conn                    |
+| application-c   | secret         | 23.4     | txenventq           | 23c-conn-string                   |
+| application-c   | secret         | 23.4     | txeventq            | 23c-kafka-name                    |
+```
+
+## Config Server REST endpoints overview
+
+The following REST Endpoints are available to Config Server entries. The table lists which minimum required role that is needed to perform the operation. `N/A` in the table below means that endpoint doesn't require authentication to be accessed.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:43]: Change to:

The following REST API endpoints are available to the Config Server entries. The table lists which minimum required role is needed to perform the operation. 'N/A' in the following table indicates that the endpoint does not require authentication to be accessed.

File: docs-source/spring/content/platform/config/_index.md [hunk 8]
```diff
@@ -1,32 +1,395 @@
...
+| End point                     | Method | Description                                             | Minimum Required Role |
+|-------------------------------|--------|---------------------------------------------------------|-----------------------|
+| /srv/config/all               | GET    | Get all distinct properties for a service (application) | N/A                   |
+| /srv/config/properties        | GET    | Get all distinct properties with filters (see examples) | N/A                   |
+| /srv/config/properties/add    | POST   | Create properties from a file                           | ROLE_USER             |
+| /srv/config/property/add      | POST   | Create a property                                       | ROLE_USER             |
+| /srv/config/property/update   | PUT    | Update a property                                       | ROLE_USER             |
+| /srv/config/properties/delete | DELETE | Delete properties with filters (see examples)           | ROLE_ADMIN            |
+
+### Config Server REST endpoints examples
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:54]: Change to:

Config Server REST API endpoint examples

File: docs-source/spring/content/platform/config/_index.md [hunk 9]
```diff
@@ -1,32 +1,395 @@
...
+
+In all examples below you need to replace `<username>:<password>` with your username and password when necessary. ([Getting User information](../../security/azn-server/)). The examples are using `curl` to interact with the REST endpoints. They also requires that you have opened a tunnel on port 8080 to either the `config-server` or `obaas-admin` service. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:56]: Change to:

In all of the following examples, replace ... necessary. ... The examples ... the REST API endpoints. This also requires an open tunnel on ...

File: docs-source/spring/content/platform/config/_index.md [hunk 10]
```diff
@@ -1,32 +1,395 @@
...
+
+```shell
+kubectl port-forward -n obaas-admin svc/obaas-admin 8080
+```
+
+#### /srv/config/all
+
+Get all distinct application services.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:64]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 11]
```diff
@@ -1,32 +1,395 @@
...
+    "name": "atael",
+    "label": "",
+    "profile": ""
+  }
+]
+```
+
+#### /srv/config/all?service-profile=\<profile-name\>
+
+Get all distinct services filtered on profile (service-profile).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:99]: Change the period to a colon.

File: docs-source/spring/content/platform/config/_index.md [hunk 12]
```diff
@@ -1,32 +1,395 @@
...
+    "name": "atael",
+    "label": "latest",
+    "profile": "dev"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>
+
+Get all properties for a service-name (application).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:120]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 13]
```diff
@@ -1,32 +1,395 @@
...
+    "value": "connectionstring-a-dev",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>
+
+Get all properties for a service-name (application) filtered on service-label (label).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:176]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 14]
```diff
@@ -1,32 +1,395 @@
...
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile) and service-profile (profile).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:213]: Change the period to a colon.

You have used "service-profile (profile)" twice in this sentence. Is that intended?

File: docs-source/spring/content/platform/config/_index.md [hunk 15]
```diff
@@ -1,32 +1,395 @@
...
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>&property-key=\<property-key\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile), service-profile (profile) and property-key (prop_key).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:251]: Change the period to a colon.

You have used "service-profile (profile)" twice in this sentence. Is that intended?

Add a comma before "and".

File: docs-source/spring/content/platform/config/_index.md [hunk 16]
```diff
@@ -1,32 +1,395 @@
...
+    "value": "23c-kafka-name",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/property/add
+
+Create a property.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:280]: Change the period to a colon.

File: docs-source/spring/content/platform/config/_index.md [hunk 17]
```diff
@@ -1,32 +1,395 @@
...
+
+```shell
+  curl -u <username>:<password> -s -X POST \
+    -d "service-name=application-d&service-label=1.0&service-profile=AI&property-key=url-to-host&property-value=hostname" \
+    http://localhost:8080/srv/config/property/add
+```
+
+Successful creation of a property returns
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:288]: Add a colon after "returns".

File: docs-source/spring/content/platform/config/_index.md [hunk 18]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property added successfully.
+```
+
+#### /srv/config/property/update
+
+Update a property.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:296]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 19]
```diff
@@ -1,32 +1,395 @@
...
+
+```shell
+curl -u <username>:<password> -s -X PUT \
+  -d "service-name=application-d&service-label=1.0&service-profile=AI&property-key=url-to-host&property-value=new-hostname" \
+   http://localhost:8080/srv/config/property/update
+```
+
+Successful update of a property returns
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:304]: Add a colon after "returns"

File: docs-source/spring/content/platform/config/_index.md [hunk 20]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property successful modified.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:307]: successful => successfully

File: docs-source/spring/content/platform/config/_index.md [hunk 21]
```diff
@@ -1,32 +1,395 @@
...
+```
+
+#### /srv/config/properties/delete?service-name\<service-name\>
+
+Delete all properties from a service (application).
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:312]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 22]
```diff
@@ -1,32 +1,395 @@
...
+
+```Shell
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=atael
+```
+
+Successful delete of properties returns:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:319]: delete => deletion

File: docs-source/spring/content/platform/config/_index.md [hunk 23]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property(ies) successfully deleted.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:322]: Property(ies) => Properties

File: docs-source/spring/content/platform/config/_index.md [hunk 24]
```diff
@@ -1,32 +1,395 @@
...
+```
+
+#### /srv/config/delete?service-profile=\<profile-name\>&service-profile=\<service-profile\>
+
+Delete all properties with a service profile.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:327]: Change the period to a colon.

File: docs-source/spring/content/platform/config/_index.md [hunk 25]
```diff
@@ -1,32 +1,395 @@
...
+
+```Shell
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=application-d& \
+  service-profile=AI
+```
+
+Successful delete of properties returns:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:335]: delete => deletion

File: docs-source/spring/content/platform/config/_index.md [hunk 26]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property(ies) successfully deleted.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:338]: Property(ies) => Properties

File: docs-source/spring/content/platform/config/_index.md [hunk 27]
```diff
@@ -1,32 +1,395 @@
...
+```
+
+#### /srv/config/delete?service-profile=\<profile-name\>&service-profile=\<service-profile\>&service-label=\<service-label\>
+
+Delete all properties from a service with a profile and a label.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:343]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 28]
```diff
@@ -1,32 +1,395 @@
...
+
+```Shell
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=application-a& \
+  service-profile=development& \
+  service-label=12c
+```
+
+Successful delete of properties returns:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:352]: delete => deletion

File: docs-source/spring/content/platform/config/_index.md [hunk 29]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property(ies) successfully deleted.
+```
+
+#### /srv/config/delete?service-profile=\<profile-name\>&service-profile=\<service-profile\>&service-label=\<service-label\>&property-key=\<property-key\>
+
+Delete all properties from a service with a profile and a label.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:360]: Change the period to a colon

File: docs-source/spring/content/platform/config/_index.md [hunk 30]
```diff
@@ -1,32 +1,395 @@
...
+
+```Shell
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=application-b& \
+  service-profile=development& \
+  service-label=23cbeta& \
+  property-key=db-dev-name
+```
+
+Successful delete of properties returns:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:370]: delete => deletion

File: docs-source/spring/content/platform/config/_index.md [hunk 31]
```diff
@@ -1,32 +1,395 @@
...
+
+```text
+Property(ies) successfully deleted.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:373]: Property(ies) => Properties

File: docs-source/spring/content/platform/config/_index.md [hunk 32]
```diff
@@ -1,32 +1,395 @@
...
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-c& \
+  service-label=23.4& \
+  service-profile=secret& \
+  property-key=txeventq
+```
+
+Example of data returned.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:261]: Change the period to a colon

**Review by rhondaday — COMMENTED** [review]
Please incorporate the suggested edits.

File: docs-source/spring/content/platform/config/_index.md (hunk 4 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:7]: Fixed

File: docs-source/spring/content/platform/config/_index.md (hunk 5 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:8]: Fixed

File: docs-source/spring/content/platform/config/_index.md (hunk 4 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:7]: I think I fixed it

File: docs-source/spring/content/platform/config/_index.md (hunk 5 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:8]: Done

File: docs-source/spring/content/platform/config/_index.md [hunk 33]
```diff
@@ -1,32 +1,395 @@
...
 
 The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+configuration in the Oracle Autonomous Database, so it easily supports labeled versions of configuration environments, as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)). 
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
+When building applications using Spring Config Server For example, the Spring Cloud Config client's Spring `application.yaml` configuration file should include to access the deployed Spring Config Server.
```
[the remark this answers is not in the document]
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:10]: Done

File: docs-source/spring/content/platform/config/_index.md (hunk 6 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:18]: Yep

File: docs-source/spring/content/platform/config/_index.md [hunk 34]
```diff
@@ -1,32 +1,395 @@
...
-      label: latest
-      profile: dev
 ```
 
-This example fetches data where the application is `atael`, profile is `dev` and the label is `latest`.
+Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table. Managing the data for the Spring Cloud Config server should be done using the CLI or via the REST API endpoints. If you prefer, you can also work directly with the `CONFIGSERVER.PROPERTIES` table in the database. How to access the database is documented here, ([Accessing the database](../../database/)).
 
-Managing the data for the Spring Cloud Config server should be done using the CLI. If you prefer, you can also work directly
-with the `CONFIGSERVER.PROPERTIES` table in the database.
+During setup of Oracle Backend for Spring Boot and Microservices the following data is loaded into `CONFIGSERVER.PROPERTIES`. This data can be deleted.
```
[the remark this answers is not in the document]
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:20]: Done

File: docs-source/spring/content/platform/config/_index.md (hunk 7 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:43]: Done

File: docs-source/spring/content/platform/config/_index.md (hunk 18 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:296]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md (hunk 20 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:307]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md (hunk 21 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:312]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md (hunk 24 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:327]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md (hunk 27 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:343]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md (hunk 29 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:360]: I need change the backend for this, stay tuned

File: docs-source/spring/content/platform/config/_index.md [hunk 35]
```diff
@@ -1,32 +1,395 @@
...
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete? \
+  service-name=application-a& \
+  service-profile=development& \
+  service-label=12c
+```
+
+Successful delete of properties returns:
+
+```text
+Property(ies) successfully deleted.
```
[the remark this answers is not in the document]
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:339]: Fixed in src code for Config Server

File: docs-source/spring/content/platform/config/_index.md (hunk 31 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:353]: Fixed in source code for config server

File: docs-source/spring/content/platform/config/_index.md (hunk 26 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:325]: Fixed in source code for Config Server

File: docs-source/spring/content/platform/config/_index.md (hunk 23 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:311]: Fixed in source code for Config Server

File: docs-source/spring/content/platform/config/_index.md (hunk 20 above)
Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md:297]: Fixed in Config Server source code

File: docs-source/spring/content/platform/config/_index.md [hunk 36]
```diff
@@ -1,32 +1,375 @@
...
 
-The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, configured to store the Configuration in the Oracle Autonomous Database to support labeled versions of configuration environments as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)).
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
+When building applications using Spring Config Server the Spring Cloud Config Client's `application.yaml` configuration file must include to access information the deployed Spring Config Server:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:9]: You need a comma after "Spring Config Server"

Change "Client's" to "client's"

Change "... include to access information the ..." to "... include access to information in the ..."

Comment (andytael) [reply docs-source/spring/content/platform/config/_index.md]: Fixed

File: docs-source/spring/content/platform/config/_index.md [hunk 37]
```diff
@@ -1,32 +1,375 @@
...
+
+### Config Server REST API endpoints examples
+
+In all of the following examples, replace `<username>:<password>` with your username and password when necessary. ([Getting User information](../../security/azn-server/)). The examples are using `curl` to interact with the REST API endpoints. This also requires an open tunnel on port 8080 to either the `config-server` or `obaas-admin` service. Use the following command to start a tunnel to the `config-server` service:
+
+```shell
+kubectl port-forward -n config-server svc/config-server 8080
+```
+
+The output will be slightly different when using a tunnel to `obaas-admin`, the data will be included in the `"body"` section.
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:61]: Change the comma to a period and init cap "The".

File: docs-source/spring/content/platform/config/_index.md [hunk 38]
```diff
@@ -1,32 +1,375 @@
...
+
+Successful delete of properties returns:
+
+```text
+Property(ies) successfully deleted.
+```
+
+## Recreate test data
+
+The config server data can be created using the following SQL Statements:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:358]: Statements => statements

**Review by rhondaday — COMMENTED** [review]
A few more comments.

File: docs-source/spring/content/platform/config/_index.md [hunk 39]
```diff
@@ -1,32 +1,375 @@
...
 
-The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, and it is configured to store the
-configuration in the Oracle Autonomous Database, so it easily supports labelled versions of configuration
-environments, as well as being accessible to a wide range of tools for managing the content.
-Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table.
+The Spring Cloud Config server is pre-configured to work with the Spring Boot Eureka service registry, configured to store the Configuration in the Oracle Autonomous Database to support labeled versions of configuration environments as well as being accessible to a wide range of tools for managing the content. More details can be found here: ([Spring Cloud Config Documentation](https://spring.io/projects/spring-cloud-config)).
 
-For example, the Spring Cloud Config client's Spring `application.yaml` configuration file could include:
+When building applications using Spring Config Server, the Spring Cloud Config client's `application.yaml` configuration file must include access information to the the deployed Spring Config Server:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:9]: Delete the extra "the"

**Review by rhondaday — COMMENTED** [review]
One more

File: docs-source/spring/content/platform/config/_index.md (hunk 20 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:297]: Okay.

File: docs-source/spring/content/platform/config/_index.md (hunk 23 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:311]: Okay.

File: docs-source/spring/content/platform/config/_index.md (hunk 26 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:325]: Okay

File: docs-source/spring/content/platform/config/_index.md (hunk 35 above)
[the remark this answers is not in the document]
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:339]: Okay

File: docs-source/spring/content/platform/config/_index.md (hunk 30 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:350]: Okay

File: docs-source/spring/content/platform/config/_index.md (hunk 31 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:353]: Okay

File: docs-source/spring/content/platform/config/_index.md [hunk 40]
```diff
@@ -1,32 +1,375 @@
...
+curl -u <username>:<password> -s -X DELETE http://localhost:8080/srv/config/properties/delete\?service-name\=application-b\&service-profile\=development\&service-label\=23cbeta\&property-key\=db-dev-name
+```
+
+Successful delete of properties returns:
+
+```text
+Property(ies) successfully deleted.
+```
+
+## Recreate test data
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:356]: Recreate => Re-create

File: docs-source/spring/content/platform/config/_index.md [hunk 41]
```diff
@@ -1,32 +1,375 @@
...
+
+The config server data can be created using the following SQL statements:
```
Comment (rhondaday) [inline docs-source/spring/content/platform/config/_index.md:358]: config server => Config Server

**Review by rhondaday — COMMENTED** [review]
Almost there!

File: docs-source/spring/content/platform/config/_index.md (hunk 30 above)
Comment (rhondaday) [reply docs-source/spring/content/platform/config/_index.md:350]: I'm sorry. I think by putting "Okay", I'm confusing the issue. Please change:

delete => deletion

**Review by rhondaday — APPROVED** [review]
Just one minor comment. My apologies for the confusion.

**Review by rhondaday — APPROVED** [review]
Looks good!

**Pull request merged**